packages/flutter/lib/src/material/expansion_panel.dart

```diff
@@ -104,6 +116,40 @@ class ExpansionPanel {
   ///
   /// Defaults to [ThemeData.cardColor].
   final Color? backgroundColor;
+
+  /// Defines the trailing icon of the panel.
+  ///
+  /// Defaults to [Icons.expand_more].
+  final IconData icon;
+
+  /// A callback which builds the expansion panel indicator to show
+  /// whether the panel is expanded or not.
+  ///
+  /// If [expandIconBuilder] is specified, then the widget it returns must
+  /// rebuild the panel with a new [isExpanded] value.
+  /// [ExpansionPanelList.expansionCallback] should still be used to update
+  /// the state correctly. A [VoidCallback] is passed as a parameter to `expandIconBuilder`
```

Comment:
These document about parameter should probably be in the `ExpansionPanelIconBuilder`